samples/spring-eventsourced-shopping-cart/src/it/java/com/example/IntegrationTest.java

```diff
@@ -40,60 +40,65 @@ public class IntegrationTest extends KalixIntegrationTestKitSupport {
 
   @Test
   public void createAndManageCart() {
-    String cartId =
-        webClient.post()
-            .uri("/cart/create")
-            .retrieve()
-            .bodyToMono(String.class)
-            .block(timeout);
+
+    String cartId = "card-abc";
+
+    ResponseEntity<String> created =
+      webClient.post()
+        .uri("/cart/" + cartId + "/create")
```

Comment:
maybe we can add a formating rule to be part of the javafmt? otherwise, we end up changing this back and forth depending on who is the next person touching the file and what settings they have on intellij...

Reply:
The problem is that samples are not part of the build. In that specific case, it's not even a sbt project. 

I'm wondering if can create some virtual sbt module that is only used to format code in the samples. As such, it doesn't need to run compilations and be aware of sample dependencies, but only run the formatting on it.